app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -303,7 +331,11 @@ fun Starter() {
                 modifier = Modifier
                     .size(18.dp)
                     .padding(2.dp)
-                    .clickable { },
+                    .clickable {
+
```

Comment:
🚫 **[ktlint]** <no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty